dev-server/documents/html/sidebar-external-container.mustache

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Annotatable iframe test</title>
+    <style>
+      body {
+        font-family: sans-serif;
+      }
+      .container {
+        display: flex;
+      }
+      #my-sidebar {
+        flex: 0 0 20%;
+      }
+      iframe {
```

Comment:
Did you intend for these rules to match both the sidebar iframe and the content iframe?

Reply:
No, `my-sidebar` is intended to occupy 20% and the rest of the space by the iframe.

Reply:
I have made a few changes to make both the sidebar and iframe resizable at the same time.